packages/appeals-service-api/src/lib/token.js

```diff
@@ -1,10 +1,12 @@
+const crypto = require('crypto');
+const TOKEN_LENGTH = 5;
+const BUFFER_LENGTH = 20;
 const createToken = () => {
-	const token = [];
-	for (let i = 0; i < 5; i += 1) {
-		const num = Math.floor(Math.random() * 9 + 1);
-		token.push(num);
-	}
-	return token.join('');
+	return crypto
+		.randomBytes(BUFFER_LENGTH)
+		.toString('base64')
+		.replace(/[^\w_]/g, '')
+		.slice(0, TOKEN_LENGTH);
```

Comment:
Is it worth picking a longer verification string as a quick way to avoid more profanities than not? According to this there are no profanities that are 9 characters long https://www.wired.com/2016/09/science-swear-words-warning-nsfw-af/
Could also allow special characters - but that may cause more problems than it's worth if we ever pass them in a url or there are any caveats in terms of queries with the mongo sdk.

Reply:
> Is it worth picking a longer verification string as a quick way to avoid more profanities than not? According to this there are no profanities that are 9 characters long https://www.wired.com/2016/09/science-swear-words-warning-nsfw-af/ Could also allow special characters - but that may cause more problems than it's worth if we ever pass them in a url or there are any caveats in terms of queries with the mongo sdk.

You could remove the vowels?